static/js/entry/payment.js

```diff
@@ -1,20 +1,29 @@
 // @flow
+import { createBrowserHistory } from "history"
```

Comment:
What do you think of changing this filename to `root.js` and updating the webpack config to reflect that?

Reply:
:+1: